File: android/src/main/kotlin/com/flutter/logs/plogs/flutter_logs/DeviceInfo.kt
package com.flutter.logs.plogs.flutter_logs

data class DeviceInfo(val osVersion: String, val appVersion: String, val deviceModel: String) {
    override fun toString(): String {
        return "\nDevice Model: $deviceModel \nOSVersion: $osVersion \nApp Version: $appVersion"
    }
}

File: ios/Classes/DeviceInfo.swift
//
//  DeviceInfo.swift
//  flutter_logs
//
//  Created by Idrish Sorathiya on 12/01/24.
//

import Foundation

public struct DeviceInfo: CustomStringConvertible {
    let osVersion: String
    let appVersion: String
    let deviceModel: String

    public var description: String {
        return "\nDevice Model: \(deviceModel) \nOS Version: \(osVersion) \nApp Version: \(appVersion)"
    }
}
